svc-lighthouse-api/src/test/java/gov/va/vro/abddataaccess/service/LighthouseApiServiceTest.java

```diff
@@ -32,41 +34,27 @@
  *
  * @author warren @Date 8/30/22
  */
+@ExtendWith(SpringExtension.class)
+@EnableConfigurationProperties
+@ContextConfiguration(classes = LighthouseProperties.class)
+@TestPropertySource({"classpath:application.properties", "classpath:application-test.properties"})
 @ExtendWith(MockitoExtension.class)
 @Slf4j
-@ActiveProfiles("test")
 class LighthouseApiServiceTest {
 
   private static final String TEST_PATIENT = "9000682";
-  private static final String CLIENTID = "00ah28rb5jLj4NeSN2p7";
-  private static final String ASSERTIONURL =
-      "https://deptva-eval.okta.com/oauth2/aus8nm1q0f7VQ0a482p7/v1/token";
-  private static final String TOKENURL = "https://sandbox-api.va.gov/oauth2/health/system/v1/token";
-  private static final String TEST_KEY_FILE = "testkey.pem";
   private static final String TEST_TOKEN = "lighthouseToken.json";
 
-  @InjectMocks private LighthouseApiService service;
```

Comment:
`@InjectMocks` not needed. Just use `new LighthouseApiService(lhProps, restTemplate)` instead.